### PR TITLE
New notebook: Load CSV tutorial

### PR DESCRIPTION
**Pull request by adammichaelwood**
b/123580430

Notebook showing loading CSV into `tf.data.Dataset`, using Titanic data.

Run it here: https://colab.research.google.com/github/adammichaelwood/tf-docs/blob/load-csv-tutorial/site/en/r2/tutorials/load_data/csv.ipynb

### CODE REVIEW

File: site/en/r2/tutorials/load_data/csv.ipynb [hunk 1]
```diff
@@ -0,0 +1,929 @@
+{
```
Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>reshuffle_each_iteration defaults to True. So you can remove it :)</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>Please use string activations.</p><p><br></p><p>Move compile out of get_model(). </p><p><br></p><p>Also, if why not use Adam() as the optimizer?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>test_loss, test_accuracy = model.evaluate(test_data)</p><p><br></p><p>print ('Test Loss {}, Test Accuracy {}'.format(test_loss, test_accuracy)</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>Don't print all the predictions.</p><p><br></p><p>The first five predictions with their real labels should convey the idea.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: Done

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: Done.
I don't know why it was not adam - @jsimsa wrote the original code.
I've changed it to "adam" now - no impact on loss.

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: Done

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: Done

File: site/en/r2/tutorials/load_data/csv.ipynb [hunk 2]
```diff
@@ -0,0 +1,933 @@
...
```
Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>Please remove the last empty line.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>You have mentioned these arguments.</p><p><br></p><p>Also, remove the arguments block, the code demonstrates it and they are pretty easy to understand based on their name.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>Remove the commented out arguments.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>Keep the shuffling where the .map() happens.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>Please use 'relu' instead of tf.keras.backend.relu.</p><p><br></p><p>Also you can do `x = tf.keras.Input(shape=(input_dim, ))`.</p><p><br></p><p>&gt; hidden_units: [int] the layer sizes of the DNN (input layer first)</p><p><br></p><p>hidden_units: (int) the layer sizes of the DNN (input layer first)</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>Please delete this sentence: "In production, you'd want to actually get the output."</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Comment (yashk2810) [inline site/en/r2/tutorials/load_data/csv.ipynb]: <p>Can you also print the real label?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: <p>Done.</p>

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: <p>Done</p>

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: <p>Done.</p>

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: <p>Done</p>

Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: <p>Done.</p><p><br></p><p>But it should be square brackets, I think, as it is a list of ints, not an int.</p>

**Comment by adammichaelwood** [comment]
@yashk2810 
I'm having trouble at the end. I tried getting the labels "manually" in Python, but the iteration doesn't work right (partly because of shuffle, and partly because of how iterators work, i think)

Is there a right way to grab the labels?

**Comment by adammichaelwood** [comment]
@yashk2810 can you look at the end and help out with your request to show the labels.

**Comment by yashk2810** [comment]
Sorry about the delay. You need to remove the shuffle from the test_data.

`test_data = raw_test_data.map(preprocess)`

The following line will give you the predictions and the corresponding labels since the data is not shuffled.

print (predictions[:5], list(test_data)[0][1][:5])

File: site/en/r2/tutorials/load_data/csv.ipynb (hunk 2 above)
Comment (adammichaelwood) [reply site/en/r2/tutorials/load_data/csv.ipynb]: i think this is right currently

**Comment by adammichaelwood** [comment]
@yashk2810 I *think* this is ready to go.
